lib/user.js

```diff
@@ -117,9 +121,22 @@ function access(permission, account, callback) {
       callback(err);
       return;
     }
-
-    callback(null, permissions.indexOf(permission) > -1);
+    findPermission(permissions);
+    callback(null, permissionFound);
+    return;
   });
+
+  function findPermission(permissions) {
```

Comment:
This could return a boolean value instead of messing with variables from different scopes